designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/InputAge.kt

```diff
@@ -53,6 +59,8 @@ import java.util.Calendar
  * @param uiModel: data class [InputAgeModel] with all parameters for component.
  * @param modifier: optional modifier.
  */
+@Suppress("DEPRECATION")
+@Deprecated("This component is deprecated and will be removed in the next release. Use InputDateTime instead.")
```

Comment:
Could you modify the reason for the deprecation?

Reply:
Done.